src/Datadog.Trace/Logging/LibLogScopeEventSubscriber.cs

```diff
@@ -39,7 +39,16 @@ public LibLogScopeEventSubscriber(IScopeManager scopeManager)
             }
             else
             {
-                SetDefaultValues();
+                // Do not set default values because the Tracer (and this event
+                // subscriber) may be initialized at a time when it is not safe
+                // to add properties to the MDC of the underlying logging framework.
+                //
+                // This previously caused a SerializationException when the Tracer
+                //  was initialized in the Asp.Net HttpModule startup, the default
+                // values were added to the underlying log4net framework, and,
+                // upon handling a new request, Asp.Net tried to copy the type
+                // log4net.Util.PropertiesDictionary from one AppDomain to another
+                // even though the type did not implement MarshalObjectByRef
```

Comment:
I think this would be a good time to prioritize getting full IIS running in CI.

Can we add a test ensuring that the values aren't there when a Tracer is initialized, and are there after a trace is started?

Reply:
I agree with getting full IIS running in CI, but I won't be making any changes there for this PR.

I will add a followup commit that adds a small unit test to make sure we don't add the properties add Tracer initialization, so stay tuned.

Reply:
Added in c9997ae7d8971244e7921cafc908469e93a5ff90